tooling/minitest/src/lib.rs

```diff
@@ -19,30 +19,30 @@ pub use std::string::String;
 mod tests;
 
 #[track_caller]
-pub fn assert_stop(prog: Program) {
-    assert_eq!(run_program(prog), TerminationInfo::MachineStop);
+pub fn assert_stop(prog: Program, model: MemoryModel) {
```

Comment:
The signature I was looking for is `fn assert_stop<M: Memory>`. No need for a new enum.

To avoid having to repeat `BasicMemory<DefaultTarget>` everywhere, you can introduce `type BasicMem = BasicMemory<DefaultTarget>;` or something like that.

Reply:
Thank you for the advice. I will fix it.